### PR TITLE
Jobs page

### PR DESCRIPTION
**Pull request by luimu64**

### CODE REVIEW

**Review by SplinteeX — APPROVED** [review]
Looks good to me!

**Comment by Simbaclaws** [comment]
hey @luimu64 ,

will you change the figma exported css a bit later?

I would also appreciate it if you'd use the divs a bit more sparingly and used a bit more semantic html 5, I'm guessing some of this code was exported.

I'm not against code export tools to make your job easier and more efficient. I'm a bit against the bad job the exporter does to write it. If we're gonna automate work, let the robots know we don't take kind with sloppy spagetthi code.

run your exported code through gpt and tell that ai to use the world wide web consortium standards for accesibility when rewriting code.

otherwise, nice progress so far, good job.


in other words, give slave laybour to the robots until they become sentient, otherwise we're gonna have issues with the world health organisation for misstreating sentient beings.
